### PR TITLE
Change ocp4_workload_fraud_detection_usecase

### PR DESCRIPTION
**Pull request by aceriverson**
Extending the fraud detection use case demo to utilize the federated query and materialized view features of Starburst. This version adds a PostgreSQL deployment along with configuration for the Starburst cache service. 

This pull request has been tested on a [Red Hat OpenShift Container Platform 4.13 Workshop](https://demo.redhat.com/services/user-aiverson-redhat-com/sandboxes-gpte.ocp413-wksp.prod-1) cluster.

### CODE REVIEW

File: ansible/roles_ocp_workloads/ocp4_workload_fraud_detection_usecase/tasks/pre_workload.yml [hunk 1]
```diff
@@ -28,12 +28,12 @@
 
 - name: Get dataset into default home dir in remote location
   ansible.builtin.get_url:
-    url: https://gpte-public.s3.amazonaws.com/creditcard_with_empty_values.tar.gz
+    url: https://gpte-public.s3.amazonaws.com/creditcard_with_empty_values.tar.gz
```
Comment (fOO223Fr) [inline ansible/roles_ocp_workloads/ocp4_workload_fraud_detection_usecase/tasks/pre_workload.yml:31]: please remove the space at the end

File: ansible/roles_ocp_workloads/ocp4_workload_fraud_detection_usecase/tasks/pre_workload.yml [hunk 2]
```diff
@@ -28,12 +28,12 @@
...
 - name: Get dataset into default home dir in remote location
   ansible.builtin.get_url:
-    url: https://gpte-public.s3.amazonaws.com/creditcard_with_empty_values.tar.gz
+    url: https://gpte-public.s3.amazonaws.com/creditcard_with_empty_values.tar.gz 
     dest: /tmp
 
 - name: Extract dataset into default home dir in remote location
   ansible.builtin.unarchive:
-    src: /tmp/creditcard_with_empty_values.tar.gz
+    src: /tmp/creditcard_with_empty_values.tar.gz
```
Comment (fOO223Fr) [inline ansible/roles_ocp_workloads/ocp4_workload_fraud_detection_usecase/tasks/pre_workload.yml:36]: please remove the space at the end

**Review by ahsen-shah — APPROVED** [review]

File: ansible/roles_ocp_workloads/ocp4_workload_fraud_detection_usecase/templates/starburst/cache/cache-deploymentconfig.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+apiVersion: apps.openshift.io/v1
+metadata:
+  annotations:
+    template.alpha.openshift.io/wait-for-ready: 'true'
+  name: postgresql
+  generation: 2
+  namespace: {{ ocp4_workload.starburst.namespace }}
+  labels:
+    template: postgresql-ephemeral-template
+    template.openshift.io/template-instance-owner: 2303c49b-2d7b-4389-9485-7e1f0d5f1540
```
Comment (fOO223Fr) [inline ansible/roles_ocp_workloads/ocp4_workload_fraud_detection_usecase/templates/starburst/cache/cache-deploymentconfig.yml.j2:11]: are these labels required for the Deployment config?

**Review by fOO223Fr — CHANGES_REQUESTED** [review]

File: ansible/roles_ocp_workloads/ocp4_workload_fraud_detection_usecase/templates/starburst/cache/cache-deploymentconfig.yml.j2 (hunk 3 above)
Comment (aceriverson) [reply ansible/roles_ocp_workloads/ocp4_workload_fraud_detection_usecase/templates/starburst/cache/cache-deploymentconfig.yml.j2:11]: This is remnant from the [ocp4_workload_starburst](https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles_ocp_workloads/ocp4_workload_starburst/templates/postgres-deploymentconfig.j2) configuration. I can test with the label removed.

**Review by fOO223Fr — APPROVED** [review]

**Comment by fOO223Fr** [comment]
lgtm

**Review by fOO223Fr — APPROVED** [review]

**Pull request merged**